### ISSUE TITLE
[Feature request]: Save Playlist with Timer

### ISSUE THREAD

**Issue by k-y-a** [vzhd1701/gridplayer#4]
### Suggestion

GridPlayer v.0.1.3

Feature Request: Save Playlist with current timer

Issue: Opening playlist again will just start from the beginning.

Reproduce: Open video, player video for a few minutes, Save Playlist. Open Playlist. Timer is not saved.

Why: So can continue watching from the playlist previously. Otherwise have to remember the current playing time, or have to finish watching in one sitting, or take screenshot and 'resume' the timer again.

---

Thank you very much for creating this software! This is pretty much the 3rd Video Player that can do this, both of the other is abandoned or rarely is updated (HAMMultiPlayer and Awesome Video Player). Looking forward to future update, I guess the only thing I can help with is if you have Translation/Multi-Language support. Will also try to bring more attention to this.

**Comment by vzhd1701** [comment]
You can already do that, actually. There is an option in settings called "Save videos position." Once checked, every playlist you save will retain video time at the moment you save it.

Thank you very much for using it! I really appreciate your feedback! :)
I will add multi-language support in the future.

**Comment by k-y-a** [comment]
> You can already do that, actually. There is an option in settings called "Save videos position." Once checked, every playlist you save will retain video time at the moment you save it.

This is what happens when I got too excited and didn't look properly. Thank you!

I post to [VideoHelp](https://forum.videohelp.com/forums/13-Software-Playing) but it under Moderation before thread get post, hopefully what I written doesn't sound like a spam post.

I also email [Ghacks](ghacks.net/), hopefully GridPlayer get covered in an article. I think it should eventually because this project is Multi-Platform and OpenSource.

Would like to thanks whoever post this on [AlternativeTo](https://alternativeto.net/software/gridplayer/about/), this how I found this.

Anyway since this post is resolve, will be closing it.

**Comment by vzhd1701** [comment]
Sorry that I didn't make a proper user manual. I will have to fix it!

Wow, thank you very, very, very much for getting the word out! I've posted it in a couple of places like softpedia and alternativeto. Also, due to this program's niche nature, I tried to seed it across the sites from the search results like "how to play multiple videos at the same time." I completely forgot about VideoHelp. Thanks again for helping out!
